### PR TITLE
Update juniperbot presence

### PR DESCRIPTION
**Pull request by MrLivixx**
- Added language selection
- Refactoring code

### CODE REVIEW

File: websites/J/Juniperbot/dist/metadata.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "en": "Fluffy Discord Bot (Music, Ranking, Reminder, Vk Community bridge, WikiFur)"
   },
   "service": "Juniperbot",
-  "version": "1.0.1",
+  "version": "1.5",
```
Comment (Bas950) [inline websites/J/Juniperbot/dist/metadata.json:17]: 1.1.0 or 2.0.0

**Review by Bas950 — CHANGES_REQUESTED** [review]
Remove all the .idea files and the yarn lock.

**Comment by MrLivixx** [comment]
Fixed it.

File: websites/J/Juniperbot/dist/metadata.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
     "en": "Fluffy Discord Bot (Music, Ranking, Reminder, Vk Community bridge, WikiFur)"
   },
   "service": "Juniperbot",
-  "version": "1.0.1",
+  "version": "2.0",
```
Comment (Bas950) [inline websites/J/Juniperbot/dist/metadata.json:17]: 2.0.0

File: websites/J/Juniperbot/presence.ts [hunk 3]
```diff
@@ -1,66 +1,138 @@
...
+    presenceData.details = (await strings).reading + (await strings).terms;
+    presenceData.smallImageKey = "list";
+    break;
+  case pathIncludes('/cookie'):
+    presenceData.details = (await strings).reading + (await strings).cookies;
+    presenceData.smallImageKey = 'list';
+    break;
+  case pathIncludes('/privacy'):
+    presenceData.details = (await strings).reading + (await strings).privacy;
+    presenceData.smallImageKey = 'list';
```
Comment (Bas950) [inline websites/J/Juniperbot/presence.ts:109]: You might want to put a space inbetween the strings

**Review by Bas950 — CHANGES_REQUESTED** [review]
Dont fully delete the yarn.lock, just undo your changes.

Upload proof of it working, (I know the new strings arent out yet, but upload proof of one that uses the general strings)

**Comment by MrLivixx** [comment]
image
Presence in Russian because there was no english :(
image

**Comment by Bas950** [comment]
> image
> Presence in Russian because there was no english :(
> image

just scroll there is always english since its default.

**Comment by MrLivixx** [comment]
> > image
> > Presence in Russian because there was no english :(
> > image
> 
> just scroll there is always english since its default.

image

File: websites/J/Juniperbot/presence.ts [hunk 4]
```diff
@@ -1,66 +1,147 @@
...
+    presenceData.details = (await strings).leaderboard;
+    presenceData.state = document.querySelector(
+      ".guild--info h1.font-weight-thin.display-2"
+    ).innerHTML;
+    presenceData.smallImageKey = "list";
+    break;
+  case pathIncludes("/dashboard/"):
+    presenceData.details = (await strings).serverdash;
+    presenceData.state = (await strings).serverdashname.replace('{name}',
+     document.querySelector(
```
Comment (Bas950) [inline websites/J/Juniperbot/presence.ts:76]: use {0} instead.

**Review by Bas950 — CHANGES_REQUESTED** [review]
After this you should be good to go

**Comment by Bas950** [comment]
FYI: waiting for api to update.

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]

**Pull request merged**